src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
![Screenshot 2022-07-18 at 08 45 04](https://user-images.githubusercontent.com/60258373/179451379-26ca7e6a-eaa5-45c1-a1e9-44a045c39595.png)

Reply:
I'm sorry. I read it wrong.